### PR TITLE
Make remove button on Suggestion visible on focus

### LINKED ISSUE #7913: Remove button on FloatingPeoplePicker is not keyboard accessible
### Environment Information 
 
- __Package version(s)__:  6.135.0

### Please provide a reproduction of the bug in a codepen:
 
https://codepen.io/adjective-object/pen/YBEQry

Focus to the button and press tab a few times. Contrast with hovering

#### Actual behavior:
 
Keyboard-only users should be able to press the Remove buttons
 
#### Expected behavior:
 
The only way to access the remove button is by using a mouse

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? Yes
 
Requested priority: Normal
 
Products/sites affected: Outlook

### PR DESCRIPTION
**Pull request by Adjective-Object**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #7913
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Because the remove button on the Suggestion item is tied to hover,
it was not keyboard accessible.

This PR makes the button appear when the adjacent item is focused,
so it can be used by keyboard-only users. (Fixes #7913).

#### Focus areas to test

FloatingPeoplePicker + Suggestions

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/7915)

### CODE REVIEW

**Comment by natalieethell** [comment]
@ecraig12345 merged some updated snapshots yesterday that should hopefully fix the travis-ci build failure. I'll close and re-open this PR to trigger a new build.

**Comment by KevinTCoughlin** [comment]
Build is passing now. @joschect can you take a look please 🙂?

File: packages/office-ui-fabric-react/src/components/pickers/Suggestions/SuggestionsItem.styles.ts [hunk 1]
```diff
@@ -28,7 +28,20 @@ export function getStyles(props: ISuggestionsItemStyleProps): ISuggestionsItemSt
           '&:hover': {
             background: palette.neutralLighter
           },
-          '&:hover .ms-Suggestions-closeButton': {
+          [`&:hover .${classNames.closeButton}`]: {
```
Comment (natalieethell) [inline packages/office-ui-fabric-react/src/components/pickers/Suggestions/SuggestionsItem.styles.ts]: Nice!

**Comment by natalieethell** [comment]
Hey @Adjective-Object thanks again so much for looking into this issue and implementing a possible solution. After speaking with @betrue-final-final, it looks like this change in behavior is going to involve a longer design discussion. We'll schedule a meeting with the team, including @joschect, to discuss the expected behavior for a focusable Close ("X") button within a suggestions item and then I think we can revisit this issue by reopening this PR or opening a new one.

**Pull request closed without merging**